reloader.js

```diff
@@ -50,7 +50,10 @@ function watch (path, rg) {
         debug('Killing worker %d', worker.id);
         worker.disconnect();
         var timeout = setTimeout(worker.kill.bind(worker), 3000);
```

Comment:
I don't think this part is actually needed. I guess we're trying to catch the case where calling `worker.disconnect()` doesn't trigger the `disconnect`-event? It seems unlikely. I'll leave it here though.. can't hurt :)